src/main/java/io/javalin/plugin/openapi/handler/ParameterHandler.kt

```diff
@@ -0,0 +1,16 @@
+package io.javalin.plugin.openapi.handler
+
+import io.javalin.http.Context
+import io.javalin.http.Handler
+import io.javalin.plugin.openapi.handler.functional.ParameterMapper
+import io.javalin.plugin.openapi.handler.functional.SerializableFunction
+
+class ParameterHandler<T : SerializableFunction>(
```

Comment:
A new interface, hidden as a "normal" handler, allowing the user to map from the Context object to their function, while providing the reference as a pointer for the annotation detection.